steampy/login.py

```diff
@@ -1,22 +1,37 @@
-import base64
-import time
-import requests
+from time import time
+from http import HTTPStatus
+from base64 import b64encode
+
+from rsa import encrypt, PublicKey
+from requests import Session, Response
+
 from steampy import guard
-import rsa
 from steampy.models import SteamUrl
-from steampy.exceptions import InvalidCredentials, CaptchaRequired
+from steampy.exceptions import InvalidCredentials, CaptchaRequired, ApiException
 
 
 class LoginExecutor:
 
-    def __init__(self, username: str, password: str, shared_secret: str, session: requests.Session) -> None:
+    def __init__(self, username: str, password: str, shared_secret: str, session: Session) -> None:
         self.username = username
         self.password = password
         self.one_time_code = ''
         self.shared_secret = shared_secret
         self.session = session
 
-    def login(self) -> requests.Session:
+    def _api_call(self, method: str, service: str, endpoint: str, version: str = 'v1', params: dict = None) -> Response:
+        url = '/'.join([SteamUrl.API_URL, service, endpoint, version])
+        # all requests from the login page use the same "Referer" and "Origin" values
+        headers = {
+            "Referer": SteamUrl.COMMUNITY_URL + '/',
+            "Origin": SteamUrl.COMMUNITY_URL
+        }
+        if method.upper() == 'GET':
+            return self.session.get(url, params=params, headers=headers)
+        else:
```

Comment:
Change it to else method.upper()  == 'PosT'

else raise some exception of not supported api call method.